### ISSUE TITLE
Support for songbird network

### ISSUE THREAD

**Issue by item333** [EthWorks/useDApp#376]
Hello, I've noticed songbird network is officially unsupported from usedapp. Is there any way I can manually add support for the network? when I swap metamask from kovan to songbird, the react app can't read the balance of the address.

**Comment by JamesGDiaz** [comment]
You can use this multicall contract I deployed, like this:
```
const config = {
  ...
  multicallAddresses: {
    19: '0x60351436cf80A31EA6C3B261C784d3C127dBD6f1',
    14: ''
  }
}
```

and then:
```
<DAppProvider config={config}>
  <App />
<DAppProvider/>
```

**Comment by tt-marek** [comment]
Fixed in #403
